custom_components/arris_dcx960/sensor.py

```diff
@@ -38,8 +38,11 @@ def unique_id(self):
 
     @property
     def name(self):
-        """Return the unique id."""
         return f"{self.username} Recording capacity"
+    
+    @property
+    def icon(self):
+        return "mdi:percent"
```

Comment:
Or:
```suggestion
        return "mdi:percent-outline"
```
Accept this if you like this better